international-journal-of-osteoarchaeology.csl

```diff
@@ -0,0 +1,248 @@
+    </group>
+  </macro>
+  <citation et-al-min="3" et-al-use-first="1" disambiguate-add-year-suffix="true" collapse="year">
+    <sort>
+      <key macro="author"/>
+      <key macro="year-date"/>
+    </sort>
+    <layout prefix="(" suffix=")" delimiter="; ">
+      <group delimiter=", ">
+        <group delimiter=", ">
+          <text macro="author-short"/>
+          <text macro="year-date"/>
+        </group>
+        <text variable="locator" prefix="p."/>
+      </group>
+    </layout>
+  </citation>
+  <bibliography et-al-min="11" et-al-use-first="1">
+    <sort>
+      <key macro="author"/>
+      <key macro="year-date"/>
+      <key variable="title"/>
+    </sort>
+    <layout>
+      <group delimiter=". " suffix=".">
+        <text macro="author"/>
+        <text macro="year-date"/>
+      </group>
+      <choose>
+        <if type="bill book graphic legal_case legislation motion_picture report song" match="any">
+          <group delimiter=". " prefix=" ">
+            <text macro="title" suffix=""/>
+            <text macro="edition"/>
+            <text macro="editor"/>
+            <text variable="genre"/>
+            <text macro="publisher" suffix="."/>
+          </group>
+        </if>
+        <else-if type="paper-conference">
+          <group delimiter=". " prefix=" ">
+            <text macro="title"/>
+            <text variable="event" prefix="presented at the "/>
+            <text variable="event-place"/>
+            <text variable="page" suffix=" pp."/>
+            <date variable="issued">
+              <date-part name="day" suffix=" "/>
+              <date-part name="month" form="long"/>
+            </date>
+          </group>
+        </else-if>
+        <else-if type="chapter paper-conference" match="any">
+          <group delimiter=" " prefix=" ">            
+            <text macro="title" suffix="."/>
+            <text macro="container" suffix="."/>
+            <text macro="publisher" suffix=";"/>
+            <text macro="refpages"/>
+          </group>
+        </else-if>
+        <else-if type="thesis">
+          <group delimiter=", " prefix=" ">
+            <text macro="title"/>
+            <text variable="genre"/>
+            <text variable="page" suffix=" pp."/>
+            <text macro="publisher"/>
+            <date variable="issued">
+              <date-part name="day" suffix=" "/>
+              <date-part name="month" form="long"/>
+            </date>
+          </group>
+        </else-if>
+        <else>
+          <group delimiter=". " prefix=" ">
+            <text macro="title"/>
+            <text macro="editor"/>
+          </group>
+          <group prefix=". ">
+            <text variable="container-title" font-style="italic"/>
+            <group prefix=" " delimiter=" ">
+              <group>
+                <text variable="volume" font-weight="bold"/>
+                <text macro="published-date"/>
+              </group>
+              <text macro="pages" prefix=" : "/>
```

Comment:
are you sure about the space before the colon? I've never seen that in English. You also have an additional space via the delimiter of the group in row 234